public-ui/src/app/highlight-search/highlight-search.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-highlight-search',
+  templateUrl: './highlight-search.component.html',
+  styleUrls: ['./highlight-search.component.css']
+})
+export class HighlightSearchComponent implements OnInit, OnChanges {
+  @Input() text: string;
+  @Input() searchTerm: string;
+  @Input() isSource: boolean;
+  @Input() conceptId: string;
+  @Input() isStandard: string;
+  @Input() indNum: number;
+  chunks: string[];
+  reString: RegExp;
+
+  constructor() { }
+
+  ngOnInit() {
```

Comment:
rm if unused (and drop the interface above)

Reply:
Done.